core/buffer.lua

```diff
@@ -357,11 +389,14 @@ text.
 function reduxbuffer:add_text(text, style, command, indicator)
   text = tostring(text)
   local insert_pos = self.target.current_pos
+  local saved_buf = buffer
+  _G.buffer = self.target
```

Comment:
@jugglerchris Could you remind me what this is for? Is the command entry behaving differently with regards to the target buffer?